content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx

```diff
@@ -10,22 +10,16 @@ import responseJson from '../_examples/response';
   [GeoIP2 Insights response body](/geoip/docs/web-services/responses/#geoip2-insights-body-example)
   with four modifications:
 
-  {/* <!--lint disable list-item-bullet-indent --> */}
-  1. `risk` has been added directly to the `ip_address` object
-  1. `local_time` has been added to the `location` sub-object
-  1. The `maxmind` object is not present. See below for descriptions.
-  1. minFraud Insights and Factors return the following anonymous IP outputs:
-      * `is_anonymous`
-      * `is_anonymous_vpn`
-      * `is_hosting_provider`
-      * `is_public_proxy`
-      * `is_residential_proxy`
-      * `is_tor_exit_node`
-  {/* <!--lint enable list-item-bullet-indent --> */}
+1. `risk` has been added directly to the `ip_address` object
+1. `local_time` has been added to the `location` sub-object
+1. The `maxmind` object is not present. See below for descriptions.
+1. minFraud Insights and Factors return the following anonymous IP outputs: _
+   `is_anonymous` _ `is_anonymous_vpn` _ `is_hosting_provider` _
+   `is_public_proxy` _ `is_residential_proxy` _ `is_tor_exit_node`
```

Comment:
This one is kinda weird. It was an unordered (sub)list, but now it is just rendered on a single line separated with underscores.

I think it should probably be changed back to a list.

Reply:
I am guessing adding a empty line before the list would prevent it from doing this. Some Markdown parsers require this.